### PR TITLE
ADD MFA TO MANAGER

### PR DESCRIPTION
**Pull request by Reinkaos**
**Acceptance Criteria**
- Sj Manager requires MFA to log in
- Document in Sj docs

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -70,4 +70,13 @@
       expect(user.run_harvest_partners).to eq ['a', 'c']
     end
   end
+
+  describe 'generate_totp' do
```
Comment (EdwinRozario) [inline spec/models/user_spec.rb:74]: `totp` or `otp`?

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -70,4 +70,13 @@
...
+    it 'generate the totp after the user has been created' do
+      user = User.new(name: 'test', password: 'password', password_confirmation: 'password', email: 'test@test.co.nz')
+      expect(user.otp_secret_key).to eq nil
+      user.save!
+      expect(user.otp_secret_key).not_to eq nil
```
Comment (EdwinRozario) [inline spec/models/user_spec.rb:79]: `expect(user.otp_secret_key).to exist`

**Review by EdwinRozario — APPROVED** [review]

File: spec/models/user_spec.rb (hunk 1 above)
Comment (Reinkaos) [reply spec/models/user_spec.rb:74]: totp :)

File: spec/models/user_spec.rb (hunk 2 above)
Comment (Reinkaos) [reply spec/models/user_spec.rb:79]: Yeah sure :D

**Review by ostigley — APPROVED** [review]
🏆 🏆 🐔 🍽️

File: lib/tasks/users.rake [hunk 3]
```diff
@@ -10,8 +10,11 @@ namespace :users do
   task configure_mfa: :environment do
     p 'Generating TOTP secret for existing users...'
     User.all.each do |user|
+      next unless user.otp_secret_key.nil?
```
Comment (paul-mesnilgrente) [inline lib/tasks/users.rake:13]: `next if user.otp_secret_key.present?` would avoid the confusing `unless`

**Pull request merged**